docs/source/packages/accounts-2fa.md

```diff
@@ -153,62 +147,39 @@ So the call of this function should look something like this:
 
 <h3 id="working-with-accounts-passwordless">Working with accounts-passwordless</h3>
 
-Following the same strategy from the previous package, you can use the function `Accounts.has2faEnabled` to verify whether or not the user has 2FA enabled. If yes, you send them their token and on next step you receive their token and their 2FA code, otherwise, you still send them their token but on the next step you don't ask them for a 2FA code.
+Following the same logic from the previous package, if the 2FA is enabled, an error will be returned to the callback of the function [`Meteor.passwordlessLoginWithToken`](https://docs.meteor.com/packages/accounts-passwordless.html#Meteor-passwordlessLoginWithToken), then you can redirect the user to a place where they can provide a code.
 
 Here it's an example:
 
-```js
-Accounts.has2faEnabled(username, (err, isEnabled) => {
-  if (err) {console.error("...", err);return;}
-  
-  Accounts.requestLoginTokenForUser({selector: "email@example.com"}, e => {
-     if (e) {console.error("...", e);return;}
-     
-     if (isEnabled) {
-      setShouldAskTokenAndCode(true);
-      return;
-     }
-
-    setShouldAskToken(true);
-  });
-});
-```
-
-Now you can either call the standard method [`Meteor.passwordlessLoginWithToken`](https://docs.meteor.com/packages/accounts-passwordless.html#Meteor-passwordlessLoginWithToken) if they don't have 2FA enabled, or in case they do, you call the method `Meteor.passwordlessLoginWithTokenAnd2faCode` that will allow you to provide a selector, token, and 2FA code:
-
-{% apibox "Meteor.passwordlessLoginWithTokenAnd2faCode" %}
-
-So, using this strategy your code should look something like this:
-
 ```js
 <button
   onClick={() => {
-    // logging in with token and code
-    if (shouldAskTokenAndCode) {
-      Meteor.passwordlessLoginWithTokenAnd2faCode(
-        email,
-        token,
-        code,
-        error => {
-          if (error) {console.error('...', error);}
-        }
-      );
-      return;
-    }
     // logging in just with token 
     Meteor.passwordlessLoginWithToken(
       email,
       token,
       error => {
-        if (error) {console.error('...', error);}
+        if (error) {
+          if (error.reason === '2FA code must be informed') {
+            // send user to a page or show a component 
+            // where they can provide a 2FA code
+            setShouldAskCode(true);
+            return;
+          }
+          console.error('...', error);
```

Comment:
Should this `console.log` be like this?

Reply:
It's an example... You can handle the errors any way you prefer.